test/e2e.go

```diff
@@ -652,6 +715,19 @@ func waitForSecret(ctx context.Context, name, namespace string) (*v1.Secret, err
 	}
 }
 
+func waitForConfigMap(ctx context.Context, name, namespace string) (*v1.ConfigMap, error) {
```

Comment:
This is an unbounded loop, i.e. if the configmap map is  never created it will spin forever

This (and the other waitFor functions) could be replaced with `client.WaitForResource(ns, "configmap", name, 60 * time.Second)`